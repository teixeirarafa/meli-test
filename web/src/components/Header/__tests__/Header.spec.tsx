import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import Header from "..";

describe("Header Component", () => {
  it("should render a Header Component", () => {
    const app = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>,
    );

    expect(app).toMatchSnapshot();
  });
});
