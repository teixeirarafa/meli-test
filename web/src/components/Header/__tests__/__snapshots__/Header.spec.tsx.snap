// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render a Header Component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="nav-header"
      >
        <div
          class="container"
        >
          <div
            class="logo"
          >
            <a
              href="/"
            >
              <img
                alt="Mercado Livre"
                src="Logo_ML.png"
                srcset="Logo_ML.png 500w, Logo_ML@2x.png.png 768w"
              />
            </a>
          </div>
          <form
            class="products-search"
          >
            <input
              placeholder="Nunca dejes de buscar"
              value=""
            />
            <button
              type="submit"
            >
              <img
                alt="Search items"
                src="ic_Search.png"
                srcset="ic_Search.png 500w, ic_Search@2x.png.png 768w"
              />
            </button>
          </form>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="nav-header"
    >
      <div
        class="container"
      >
        <div
          class="logo"
        >
          <a
            href="/"
          >
            <img
              alt="Mercado Livre"
              src="Logo_ML.png"
              srcset="Logo_ML.png 500w, Logo_ML@2x.png.png 768w"
            />
          </a>
        </div>
        <form
          class="products-search"
        >
          <input
            placeholder="Nunca dejes de buscar"
            value=""
          />
          <button
            type="submit"
          >
            <img
              alt="Search items"
              src="ic_Search.png"
              srcset="ic_Search.png 500w, ic_Search@2x.png.png 768w"
            />
          </button>
        </form>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
